tools/sdk-testgen/packages/autorest.testmodeler/README.md

```diff
@@ -7,7 +7,7 @@ https://github.com/Azure/azure-rest-api-specs
 ## How to Generate Test Model
 
 ```
-autorest --version=3.7.3 --use=<test model extension> --output-folder=<RP package path> --testmodeler.export-codemodel --clear-output-folder=false --debug <RP config md file path>
+autorest --version=3.7.3 --use=<test model extension> --output-folder=<RP package path> --testmodeler.export-codemodel --clear-output-folder=false --debug <RP config md file path> --testmodeler.debugger
```

Comment:
was the debugger added during testing, or we intentionally want it there

Reply:
Removed it.